Rocket.Chat/Resources/es.lproj/Localizable.strings

```diff
@@ -1,228 +1,229 @@
 // Global
-"global.ok" = "OK";
+"global.ok" = "De Acuerdo";
```

Comment:
Do you think we really need to change the "OK" string? It would be nice to keep it short.

Reply:
fixed